install/kubernetes/quick-install.yaml

```diff
@@ -463,6 +454,7 @@ spec:
         - mountPath: /var/run/cilium
           name: cilium-run
       restartPolicy: Always
+      priorityClassName: system-node-critical
```

Comment:
I don't think this should be here - I recall that this got added when running `make` in the `install/kubernetes` directory when I was releasing a micro-release of `v1.6` and @tgraf said it shouldn't be there - @tgraf can you confirm?